### PR TITLE
allow "imac" cpu variant

### PR DESCRIPTION
**Pull request by dayjaby**
I do most my tests with cpu-variant=imac+debug. This makes it possible to run the baremetal demo also in Renode with this CPU variant.

### CODE REVIEW

**Comment by dayjaby** [comment]
Changed it, so that all CPU variants currently described in LiteX for vexriscv and vexriscv_smp should be handled.

File: generate-renode-scripts.py [hunk 1]
```diff
@@ -210,6 +210,10 @@ def generate_cpu(time_provider):
     cpuType: "rv32ima"
     privilegeArchitecture: PrivilegeArchitecture.Priv1_10
 """
+        elif variant == in ["i", "im", "ima", "imac"]:
```
Comment (mateusz-holenko) [inline generate-renode-scripts.py]: is this correct `== in`? Shouldn't it be just `variant in [...]`?

**Review by mateusz-holenko — COMMENTED** [review]
@dayjaby Thanks for your contribution!

Apart from one syntax question, the idea looks good to me.

File: generate-renode-scripts.py (hunk 1 above)
Comment (dayjaby) [reply generate-renode-scripts.py]: Of course just "in". Pushed the correct one now.

**Comment by mateusz-holenko** [comment]
Thanks, merged!

**Pull request merged**